clustering.ipynb

```diff
@@ -64,6 +64,8 @@
     "import pandas as pd\n",
```

Comment:
<p>Can you please clarify <strong>in code implementation</strong> how did you infer that (data['is_participant'] = 0) doesn't help?</p><p>To show the flow and reasons behind dropping these rows, e.g., the rows for most of the features are NULLs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done